docs/version-1.9/modules/en/pages/installation-setup/installation/installation.adoc

```diff
@@ -9,7 +9,6 @@ Please see xref:installation-setup/best-practices.adoc[Best Practices] for how t
 {longhorn-product-name} can be installed on a Kubernetes cluster in several ways:
 
```

Comment:
For consistency

1. Change `air gapped` to `air-gapped`
2. Change `Air Gap Installation` to `Air-Gapped Installation`

Reply:
@sushant-suse, make sure to apply the same changes to the upstream document.